scripts/release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status
+
+release_tag=$(echo "$1" | xargs)  # Trim whitespace
+sha=$(echo "$2" | xargs)  # Trim whitespace
```

Comment:
How is `sha` gen'd?

Reply:
Ah, commit `sha`.  Yeah?

Reply:
Yeah, wanted to keep it simple for now and let user specify the commit SHA to tag. I didn't want to assume that the release is being cut right after a merge that updates the version. Perhaps this can be improved upon in the future.